www/about/policies/terms-of-service.md

```diff
@@ -35,94 +35,97 @@ or damage arising from your failure to keep your account secure.
 your legal guardian must consent to your use and must agree to be bound by
 these Terms.
 
-1. The Gratipay Model and General Rules
+1. General Terms
 
-  1. The Service is a platform to enable Teams of Gratipay Participants to
-     receive payments to fund Open Work. Open Work means that the Team provides
-a clear path for any individual to voluntarily begin contributing to the Team's
-work and to share in any revenue the Team generates. Some examples of Teams
-performing Open Work include an open source software project that pays
-contributors, or a hackerspace that pays individuals to teach classes or manage
-its operations.
+  1. The Service is a platform to facilitate payments for goods or services
+     ("Work").
 
-  1. The Participant who establishes a Team is its Owner. Teams may optionally
-     have other Members.
+  1. A Participant may establish a Project on the Service. The Participant who
+     establishes a Project is the Project Owner.
 
-  1. When you establish a Team on the Service to fund your Open Work, you agree
-     that payments will be used exclusively to support the Open Work described
-in your Team's profile.
+  1. Projects may optionally have other Participants as Collaborators. The
+     Project Owner is also a Collaborator.
 
-  1. When you make a payment to a Team, it is a payment to fund the Open Work
-     the Team provides. You understand and acknowledge that your payment does
-not entitle you personally to receive any goods or services from the Team,
-unless the Team explicitly offers those things to Participants.
+  1. A Participant may make payments to a Project Owner via a Project. A
+     Participant who makes payments via a Project is a Giver.
 
-  1. By accepting a payment from a Participant, a Team forms an agreement with
-     that Participant to perform the Open Work described in its profile. The
-Service is a platform to facilitate this agreement. Gratipay is not a party to
-the agreement, and is not responsible to either the Team or the Participant if
-either party breaches the terms of the agreement (e.g. if the Team fails to
-provide the promised services).
+1. Terms for Project Owners
```

Comment:
Section 3 should probably have a parallel to Section 4.ii. (Something like "Project Owners may only make payments to a Collaborator for their contributions...")